### PR TITLE
Update some links through the plugin to properly attribute them

### PR DESCRIPTION
**Pull request by pattonwebz**
[PRO-165]

This pull request introduces improvements to link generation and updates existing links across the codebase to use a new UTM parameter helper function. The most significant changes include the addition of the `edac_simple_utm_link_wrapper` function for simplified UTM link creation and the replacement of hardcoded links with dynamically generated ones using this helper function.

### Link Generation Enhancements:
* **Addition of `edac_simple_utm_link_wrapper` function**: A new helper function was added to simplify the creation of links with UTM parameters. It offers flexibility to echo or return the generated link and integrates with the existing `edac_generate_link_type` function. (`includes/helper-functions.php`, [includes/helper-functions.phpR625-R663](diffhunk://#diff-862b071e8ce1ac2c78e1ccbe61ddbdc8f5b078945007bf5e6421695bed88d910R625-R663))

### Updates to Existing Links:
* **Dynamic link generation in settings page callbacks**: Hardcoded links in settings page callbacks (`edac_general_cb`, `edac_simplified_summary_cb`, `edac_post_types_cb`) were replaced with dynamically generated links using `edac_simple_utm_link_wrapper` or `edac_generate_link_type`. (`includes/options-page.php`, [[1]](diffhunk://#diff-92fd73fa85986ecef05555786d999392e201dd161e2c0e63b1e0bd3a139ce954L245-R261) [[2]](diffhunk://#diff-92fd73fa85986ecef05555786d999392e201dd161e2c0e63b1e0bd3a139ce954L277-R295) [[3]](diffhunk://#diff-92fd73fa85986ecef05555786d999392e201dd161e2c0e63b1e0bd3a139ce954L448-R466)
* **Dynamic link generation in welcome page**: Multiple hardcoded links in the welcome page were updated to use the `edac_simple_utm_link_wrapper` function for consistency and easier UTM parameter management. (`partials/welcome-page.php`, [[1]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L39-R39) [[2]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L73-R73) [[3]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L83-R119) [[4]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L138-R138) [[5]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L160-R167) [[6]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L175-R179) [[7]](diffhunk://#diff-99440eba5616a14f4b27ad1249a66fad6d99b6877b97e1a93b0c4ddbc5f64335L211-R214)

### Content Updates:
* **Improved disclaimer rendering in summary generation**: The disclaimer in `summary()` was updated to use `edac_generate_link_type` for link creation, ensuring consistency with other changes. (`admin/class-ajax.php`, [admin/class-ajax.phpL168-R184](diffhunk://#diff-87159e3bf5e81593c7698827c5dc895f087a1766d135de0dd7588b340903404eL168-R184))

### Minor Adjustments:
* **Simplified UTM parameters in accessibility statement**: The link in `get_accessibility_statement()` was updated to use fewer UTM parameters for clarity. (`includes/classes/class-accessibility-statement.php`, [includes/classes/class-accessibility-statement.phpR47-R49](diffhunk://#diff-33fcd7a62cc456354b07b91190c766276316adada327d0e59843e7f9c34fa406R47-R49))

## Checklist

- [ ] PR is linked to the main issue in the repo
- [ ] Tests are added that cover changes

## Summary by CodeRabbit

* **New Features**
  * External links throughout the plugin now include dynamic tracking parameters for improved consistency and analytics.
  * Added new localized error messages for "An error occurred while clearing issues" in multiple languages.

* **Bug Fixes**
  * Improved input validation for link generation functions to prevent errors with invalid parameters.

* **Style**
  * Updated translation strings in various languages for improved clarity, tone, and accuracy.

* **Documentation**
  * Enhanced translation template with updated formatting flags and translator comments for better localization support.

* **Chores**
  * Updated translation files to reflect the latest WP-CLI version.

### CODE REVIEW

File: partials/welcome-page.php [hunk 1]
```diff
@@ -208,7 +211,7 @@
 				</ul>
 			</div>
 			<div class="edac-pro-callout-button--wrapper">
-				<a class="edac-pro-callout-button" href="https://equalizedigital.com/accessibility-checker/pricing/?utm_source=accessibility-checker&utm_medium=software&utm_campaign=welcome-page" target="_blank">
+				<a class="edac-pro-callout-button" href="<?php edac_simple_utm_link_wrapper( 'https://equalizedigital.com/accessibility-checker/pricing/', 'welcome-page', 'get-pro-cta' ); ?>" target="_blank">
```
[the remark this answers is not in the document]
Comment (pattonwebz) [reply partials/welcome-page.php:214]: When this echoes it outputs through esc_url already
